src/Microsoft.ML.Recommender/MatrixFactorizationPredictor.cs

```diff
@@ -267,9 +277,22 @@ private void MapperCore(in uint srcCol, ref uint srcRow, ref float dst)
                 dst = float.NaN;
                 return;
             }
+
+            // The index system in the LIBMF (the library trains the model) is 0-based, so we need to deduct one
+            // from 1-based indexes returned by ML.NET's key-valued getters. We also throw when seeing 0 becuase
+            // missing index is not meaningful to the trained model.
             dst = Score((int)(srcCol - 1), (int)(srcRow - 1));
         }
 
+        /// <summary>
+        /// Compute the (approximated) value at the <paramref name="columnIndex"/>-th column and the
+        /// <paramref name="rowIndex"/>-th row. Notice that, in contrast to <see cref="MapperCore"/>,
+        /// both of <paramref name="columnIndex"/> and <paramref name="rowIndex"/> are 0-based indexes,
+        /// so the first row/column index is 0.
+        /// </summary>
+        /// <param name="columnIndex">0-based column index.</param>
+        /// <param name="rowIndex">0-based row index.</param>
+        /// <returns></returns>
```

Comment:
empty tags might cause warnings on the docs engine. please either fill it or remove it.